apps/src/templates/PaneHeader.jsx

```diff
@@ -192,10 +192,24 @@ export const PaneButton = Radium(function(props) {
     }
   }
 
+  function onKeyDownWrapper(event) {
+    const {onClick} = props;
+    if (
+      event.key === ' ' ||
+      event.key === 'Enter' ||
+      event.key === 'Spacebar'
+    ) {
+      onClick();
+    }
+  }
+
   return (
     <div
+      role="button"
+      tabIndex={0}
```

Comment:
Nit - I don't think this needs to be an object, I think it can just be a string - "0"